core/crypto/bls_provider/libbls_signatures.h

```diff
@@ -0,0 +1,126 @@
+/* libbls_signatures Header Version 0.1.0 */
+
```

Comment:
This file is generated while compiling Rust implementation of the BLS signatures and we don't need to copy it here except one reason: we can't suppress clang-tidy warning about "Include header not found".
Also add Rust toolchain to the clang-tidy CI/CD and build project again isn't a good decision.